core/project/Dependencies.scala

```diff
@@ -22,6 +22,8 @@ object Library {
   val AkkaStreamContrib     = "com.typesafe.akka"     %% "akka-stream-contrib"       % "0.10"
   val AkkaStreamKafka       = "com.typesafe.akka"     %% "akka-stream-kafka"         % Version.AlpakkaKafka exclude("com.fasterxml.jackson.core","jackson-databind") exclude("com.fasterxml.jackson.module", "jackson-module-scala")
   val AkkaStreamKafkaTestkit = "com.typesafe.akka"    %% "akka-stream-kafka-testkit" % Version.AlpakkaKafka exclude("com.typesafe.akka", "akka-stream-testkit")
+  val AkkaStreamTestkit      = "com.typesafe.akka"   %% "akka-stream-testkit"        % Version.Akka         
+  
   val AkkaCluster           = "com.typesafe.akka"     %% "akka-cluster"              % Version.Akka
```

Comment:
Artery TCP is now default, so the https://github.com/lightbend/cloudflow/blob/master/core/cloudflow-akka/src/main/resources/akka-cluster-k8.conf#L32 should be updated to use Artery config. See https://doc.akka.io/docs/akka/current/remoting-artery.html#configuration

Reply:
also, if you have any `akka.tcp` addresses they should be replaced with `akka`. see details in migration guide https://doc.akka.io/docs/akka/current/project/migration-guide-2.5.x-2.6.x.html#migrating-from-classic-remoting-to-artery

Reply:
Thanks Patrik!

Reply:
Fixed.